bitrise.yml

```diff
@@ -2,42 +2,6 @@ format_version: "7"
 default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
 
 workflows:
-  # ----------------------------------------------------------------
-  # --- workflows for CI and testing
-  ci-docker:
-    steps:
-    - script:
-        inputs:
-        - content: |-
-            #!/bin/env bash
-            set -ex
-            docker-compose build
-            docker-compose run --rm app bitrise run ci
-  maintenance:
-    steps:
-    - script:
-        title: Check step version change
-        inputs:
-        - content: |-
-            #!/usr/bin/env bash
-            go get -u github.com/godrei/stepper
-
-            # explanation:
-            # 1. run stepper tool
-            # 2. extract relevant part
-            # https://stackoverflow.com/questions/7103531/how-to-get-the-part-of-a-file-after-the-first-line-that-matches-a-regular-expres
-            stepper stepLatests --steps-const-file=$BITRISE_SOURCE_DIR/steps/const.go | sed -n -e '/package steps/,$p' > stepper_output.go
```

Comment:
the stepper tool seems still useful, should we preserve it?

Reply:
I think you are right. I will add it back. 

(Although I have not used this stepper tool so far. Maybe I should have. 😄)